src/bumper.js

```diff
@@ -520,13 +528,14 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
     return new AdBreak({type, position: this._adBreakPosition, numAds: 1});
   }
 
-  _getAdError(): Error {
+  _getAdError(promiseError: ?MediaError): Error {
```

Comment:
I would change the name promiseError in here, the method doesn’t care where the error originated, same goes for onError method

Reply:
Fixed here. in `onError` I keep it to hint it's autoplay failure